docroot/modules/custom/mass_entity_usage/tests/src/ExistingSite/EntityUsageTest.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace Drupal\Tests\mass_entity_usage\ExistingSite;
+
+use Drupal\file\Entity\File;
+use Drupal\mass_content_moderation\MassModeration;
+use Drupal\user\Entity\User;
+use weitzman\DrupalTestTraits\Entity\MediaCreationTrait;
+use weitzman\DrupalTestTraits\ExistingSiteBase;
+use weitzman\LoginTrait\LoginTrait;
+
+/**
+ * Class EntityUsageTest.
+ */
+class EntityUsageTest extends ExistingSiteBase {
+
+  use LoginTrait;
+  use MediaCreationTrait;
+
+  private $user;
+
+  /**
+   * Create the user.
+   */
+  protected function setUp() {
+    parent::setUp();
+    $user = User::create(['name' => $this->randomMachineName()]);
+    $user->addRole('administrator');
```

Comment:
Can we change this to "Content administrator" and "tester" role actually so more can test it?

Reply:
This is test code so it wont affect anything that a web user would do. Does that answer your request?

Reply:
Yes